db/migrate/20161102160102_add_trade_reporting_unpermitted_field_to_organisations.rb

```diff
@@ -0,0 +1,5 @@
+class AddTradeReportingUnpermittedFieldToOrganisations < ActiveRecord::Migration[5.0]
+  def change
+    add_column :organisations, :trade_reporting_unpermitted, :boolean, default: false
```

Comment:
Sorry I'm being a pain, but it seems to me it would be clearer if this setting read sth like" trade_reporting_enabled". All the places where we check for `!unpermitted?` would then become `enabled?` which feels much easier to process when you read the code. :smiley: Also I think the default behaviour should be that it is disabled - in this way by default Parties need to opt in, rather have to opt out.